content/pl_docker/docker_course_dev.md

```diff
@@ -188,6 +188,11 @@ You're ready to develop questions for the `OPB`!
 
 For development of questions that require an autograder, you will need to set up a local autograder container as well. 
 
+```{important}
+Follow this section for any process that requires usage of 'Open Workspace'. Some questions, such as the example below, contain built-in notebooks. To view these questions, follow the docker initialization command in this section instead of the command described in step 8.
```

Comment:
```suggestion
Follow this section for any questions that require usage of external autograders, including PrairieLearn workspace questions. 

Some questions, such as the example below, contain built-in notebooks. To work with these questions locally, follow the docker initialization command in this section instead of the command described in step 8.
```